3.Tests/UI/StepDefinitions/CommonStepDefinition.cs

```diff
@@ -95,7 +95,7 @@ public void Type(string input, string elementName, string viewName)
         }
     }
 
-    [When(@"(?:the user )?opens the context menu on <([a-zA-Z ]+)>(?: on '([a-zA-Z ]+)')?$")]
+    [When(@"(?:the user )?opens the Context Menu on <([a-zA-Z ]+)>(?: on '([a-zA-Z ]+)')?$")]
```

Comment:
Does this work for items as well?

Reply:
After seeing this question I refactored it to work with the items as well but it has some problems, we have 'priority' in the item context menu that has a different unique selector for it, and also the 'delete' option doesn't work because it has an additional class for the items.

Reply:
Valeria and I implemented a 2-step way to do this and those steps are in the CommonStepDefinition.cs file.
We first hovers over the item and the clicks on the element in the context menu.
I think you can use those steps and they should work fine.

Here is an example:
When the user hovers on "Get Item" <Bananas> on 'Items Component'
And the user clicks on "Item Contextmenu" <Bananas> on "Delete" option on "Items Component"